### PR TITLE
 add filter to image according to keyword

### PR DESCRIPTION
**Pull request by i053322**

### CODE REVIEW

File: src/edit-image.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ export default function EditImage(
                     onClick={() => inputEl?.current?.click()} />}
             </div>
             <div>מילות חיפוש</div>
-            <input type="text" value={keywords} onChange={(e) => { }} />
+            <input type="text" value={keywords} onChange={(e) => { setKeywords([e.target.value]) }} />
```
Comment (ariel-bentu) [inline src/edit-image.tsx:161]: if keywords is an array of strings, this seems wrong. as you just put as the first element all keywords?

File: src/api.tsx [hunk 2]
```diff
@@ -406,16 +407,17 @@ export async function deleteEvent(id: string, deleteModifiedInstance: boolean =
     return [];
 }
 
-export async function updateMediaInfo(imageInfo: ImageInfo) {
+export async function updateMediaInfo(imageInfo: MediaResource) {
     if (imageInfo._ref) {
+        const keywords = imageInfo.keywords?.toString().split(",");
```
Comment (ariel-bentu) [inline src/api.tsx:412]: how does the user know that keywords are separated by commas?
I would expect some text saying this, or maybe also be tolerant to spaces?
also, maybe trim spaces of the keywords?

File: src/edit-image.tsx (hunk 1 above)
Comment (i053322) [reply src/edit-image.tsx:161]: it is saved as array of string in media.tx line 15 I do the split
 const _save = useCallback((imageInfo: MediaResource, file?: File) => {
so when the user inserted the data it is array with  only 1 element "oneley,secondkey..." and when I do the save I do the split

File: src/api.tsx (hunk 2 above)
Comment (i053322) [reply src/api.tsx:412]: fixed

Comment (i053322) [reply src/api.tsx:412]: added trim and change the delimiter to space

**Pull request merged**